plugins/inventory/terraform.py

```diff
@@ -75,6 +75,20 @@ def inner(*args, **kwargs):
         if attrs.get('publicly_routable', False):
             groups.append('publicly_routable')
 
+        groups.append(attrs['consul_dc'])  # datacenter
```

Comment:
this and the groups below should be removed. They're not necessary in the new scheme.